open_academy/models/session.py

```diff
@@ -10,20 +10,24 @@ class Session(models.Model):
     active = fields.Boolean(default=True)
     duration = fields.Float()
     number_of_seats = fields.Integer()
-    instructor_id = fields.Many2one('res.partner', domain="['|',('is_instructor', '=', True),('category_id.name', 'like', 'Teacher')]")
+    instructor_id = fields.Many2one('res.partner', domain="['|',"
+                                    "('is_instructor', '=', True),"
+                                    "('category_id.name', 'like', 'Teacher')]")
```

Comment:
This is more readable

```suggestion
instructor_id = fields.Many2one(
        "res.partner", domain="['|', ('is_instructor', '=', True), ('category_id.name', 'like', 'Teacher')]"
    )
```